portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,52 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Sharon Dong</title>
     <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Work Sans' rel='stylesheet'>    
```

Comment:
Html formatting isn't necessary now but during the week 6 project, we should follow the html/css guidelines.

https://engdoc.corp.google.com/eng/doc/devguide/htmlcss/styleguide.md?cl=head#HTML_Quotation_Marks

Reply:
Oops I missed that one, thanks! I'll just change it now.